components/script/dom/document.rs

```diff
@@ -115,7 +115,7 @@ use style::servo::Stylesheet;
 use task_source::dom_manipulation::DOMManipulationTask;
 use time;
 use url::percent_encoding::percent_decode;
-use url::{Host, Url};
```

Comment:
This is already fixed in #10365; could you please revert this? (`commit --amend` to avoid creating a separate commit).